src/Views/FirmwareView.vala

```diff
@@ -0,0 +1,117 @@
+/*
+* Copyright (c) 2020 elementary, Inc. (https://elementary.io)
+*
+* This program is free software; you can redistribute it and/or
+* modify it under the terms of the GNU General Public
+* License as published by the Free Software Foundation; either
+* version 3 of the License, or (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+* General Public License for more details.
+*
+* You should have received a copy of the GNU General Public
+* License along with this program; if not, write to the
+* Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+* Boston, MA 02110-1301 USA
+*
+* Authored by: Marius Meisenzahl <mariusmeisenzahl@gmail.com>
+*/
+
+public class About.FirmwareView : Granite.SettingsPage {
```

Comment:
is there a reason not to use `SimpleSettingsPage`? That would handle all the title area creation for you

Reply:
I'm actually wondering if we even need the title and icon… we have the tab right above with "Firmware", and this view looks fine with just a list of devices.

![Screenshot from 2021-01-14 13 28 41](https://user-images.githubusercontent.com/611168/104645772-c1a1df80-566c-11eb-944f-5789e74e2e58.png)